components/table/__tests__/__snapshots__/Table.expand.test.js.snap

```diff
@@ -57,7 +57,7 @@ exports[`Table.expand click to expand 1`] = `
                 </tr>
                 <tr
                   class="ant-table-row ant-table-row-level-1"
-                  data-row-key="2"
+                  data-row-key="1"
```

Comment:
What's the reason for these changes? I saw the CI on master is green. Do you have the same problem when running `yarn test` locally?